libgcc/config.host

```diff
@@ -459,29 +459,31 @@ aarch64*-*-vxworks7*)
 aarch64-*-mingw*)
 	case ${target_thread_file} in
 	  win32)
-	    tmake_thr_file="i386/t-gthr-win32"
-	    ;;
+		tmake_thr_file="i386/t-gthr-win32"
+		;;
 	  posix)
-	    tmake_thr_file="i386/t-mingw-pthread"
-	    ;;
+		tmake_thr_file="i386/t-mingw-pthread"
+		;;
+	  mcf)
+		tmake_thr_file="i386/t-mingw-mcfgthread"
+		;;
 	esac
-	# This has to match the logic for DWARF2_UNWIND_INFO in gcc/config/i386/cygming.h
+	# This has to match the logic for DWARF2_UNWIND_INFO in gcc/config/aarch64/cygming.h
 	if test x$ac_cv_sjlj_exceptions = xyes; then
-		tmake_eh_file="i386/t-sjlj-eh"
+	  tmake_eh_file="i386/t-sjlj-eh"
 	else
-		tmake_eh_file="i386/t-seh-eh"
+	  tmake_eh_file="i386/t-seh-eh"
 	fi
 	# Shared libgcc DLL install dir depends on cross/native build.
 	if test x${build} = x${host} ; then
 	  tmake_dlldir_file="i386/t-dlldir"
 	else
 	  tmake_dlldir_file="i386/t-dlldir-x"
 	fi
-	tmake_file="${tmake_file} ${tmake_eh_file} ${tmake_thr_file} ${tmake_dlldir_file}" # i386/t-chkstk
-        tmake_file="${tmake_file} i386/t-slibgcc-cygming i386/t-slibgcc-mingw i386/t-cygming i386/t-mingw32 t-dfprules"
+	tmake_file="${tmake_file} ${tmake_eh_file} ${tmake_thr_file} ${tmake_dlldir_file} i386/t-slibgcc-cygming i386/t-slibgcc-mingw i386/t-cygming i386/t-mingw32 t-dfprules t-crtfm" # aarch64/t-chkstk
 	tmake_file="${tmake_file} ${cpu_type}/t-aarch64"
-	tmake_file="${tmake_file} ${cpu_type}/t-lse t-slibgcc-libgcc"
-	tmake_file="${tmake_file} ${cpu_type}/t-softfp t-softfp t-crtfm"
+	tmake_file="${tmake_file} ${cpu_type}/t-lse"
+	tmake_file="${tmake_file} ${cpu_type}/t-sme ${cpu_type}/t-softfp t-softfp"
```

Comment:
t-crtfm?

Reply:
One resolves to `config/aarch64/t-softfp` the other to `config/t-softfp`.

Reply:
Moved to #15.